samples/wcfapp/README.md

```diff
@@ -0,0 +1,140 @@
+172.23.70.146
+```
+### Build a Container with Self-hosted WCF Service
+Project `WcfServiceConsoleApp` is created from Windows Classic Desktop 'Console App' template in Visual Studio. We added a [Dockerfile](/Dockerfile.console) to the project. We use `microsoft/dotnet-framework` image as the base image and expose ports 80 (for HTTP) and 808 (for NET.TCP) for the container.
+
+Run commands below to build the container image with name `selfhostedwcfservice` and start an instance of it named `myservice2`.
+```
+docker build --pull -t selfhostedwcfservice -f Dockerfile.console .
+docker run -it --rm --name myservice2 selfhostedwcfservice
+```
+Open another console window to find the IP address of the self-hosted WCF service (alternatively, you can run the self-hosted WCF service container in detached mode by `docker run -d --rm --name myservice2 selfhostedwcfservice`).
+```
+docker inspect --format="{{.NetworkSettings.Networks.nat.IPAddress}}" myservice2
+172.23.69.75
+```
+### Build a Container to run WCF Client against the Service
+Now that we have WCF services running in containers. Let's run the WCF client against them. Run commands below to build the container image with name `wcfclient`. The IP address of the WCF service is passed through the environment variable `host`.
+```
+docker build --pull -t wcfclient -f Dockerfile.client .
+docker run -it --rm -e HOST=172.23.70.146 --name myclient wcfclient
+Client OS: Microsoft Windows NT 6.2.9200.0
+Service Host: 172.23.70.146
+Hello WCF via Http from Container!
+Hello WCF via Net.Tcp from Container!
+```
+The result above is from running the WCF client against the IIS-hosted WCF service. Changing the environment variable `host` to the IP address of the self-hosted WCF service, you will get a similar result.
+
+*To run WCF client on a different machine other than the one that hosts container instance*, we will need to map ports of a container to ports of its host by using Docker parameter `-p` when we start a constainer instance. For example, the command below started an instance of the self-hosted WCF service container named `myservice3` with its ports 80 and 808 mapped to ports 80 and 808 of its host machine respectively.
+```
+C:\wcfapp\WcfServiceConsoleApp>docker run -d -p 80:80 -p 808:808 --name myservice3 selfhostedwcfservice
+```
+Then for the WCF client to connect to the service, we need to set the `host` to be the IP address (or DNS name) of the container host machine (instead of the IP address of the container instance). The rest will be the same to start the WCF client.
+
+## Build and run the sample with Docker Compose
+[Docker Compose](https://docs.docker.com/compose/overview/) is a tool for defining and running multi-container Docker applications. In this sample, we also added YML files to configure WCF server and client applications. You can directly start and run service and client applications without finding IP address of service container.
```

Comment:
What do you think about changing `without finding IP address of service container` to `without having to any work to hookup the client to the WCF service"